apps/meteor/client/components/GenericNoResults/GenericNoResults.tsx

```diff
@@ -9,22 +9,40 @@ type GenericNoResultsProps = {
 	description?: string;
 	buttonTitle?: string;
 	buttonAction?: () => void;
+	linkText?: string;
+	linkHref?: string;
```

Comment:
Could we use a union instead of optionals here? E.g. it would prevent passing `linkText` without `linkHref`.